pkg/localmetrics/localmetrics.go

```diff
@@ -97,7 +97,15 @@ func UpdateAccountCRMetrics(accountList *awsv1alpha1.AccountList) {
 	reuseAccountFailedCount := 0
 	pendingVerificationAccountCount := 0
 	readyAccountCount := 0
+	idMap := make(map[string]int)
 	for _, account := range accountList.Items {
+		if idMap[account.Spec.LegalEntity.ID] == 0 {
```

Comment:
Not sure how Go does this, but I think you need to check if the key doesn't exist and create it.

Reply:
I think that the value is equal to 0 by default if the key doesn't exist so that is why i did if the val == 0, and then i think you can just set the key = to the value like in this example? https://blog.golang.org/go-maps-in-action

Reply:
Thanks for the docs, this looks right then.